docs/tutorial/bridging-ft-token.md

```diff
@@ -0,0 +1,253 @@
+
+After setting the environmental variables, you can access them in following commands.
+
+## Step 9: Registering the Account from the Wallet
+
+To register the account from your wallet, run the following code:
+
+```bash
+near call wrap.testnet storage_deposit '{"account_id": "$REGISTER_ACCOUNT_ID"}' --accountId $CALLER_ACCOUNT_ID --amount 0.00125
+```
+
+### Registering the FT Connector
+
+The FT Connector acts as a locker. From the **FT Connector** section, click on **Calimero Contracts**.
+
+To register the FT Connector, run the following code:
+
+```bash
+near call wrap.testnet storage_deposit '{"account_id": "ft_connector.$SHARD_ID.calimero.testnet"}' --accountId $CALLER_ACCOUNT_ID --amount 0.00125
+```
+
+### Registering FT token
+
+-  From the FT Connector, click on the **Register FT** button
+
+<img width="1172" alt="registerfttoken" src="https://github.com/calimero-is-near/docs/assets/39309699/eead3441-b373-47e5-b017-952e9c0a3322"/>
+
+-  Enter your FT contract address (in our case, it's `wrap.testnet`)
+-  Click on the **Register** button
+
+<img width="430" alt="contractaddress" src="https://github.com/calimero-is-near/docs/assets/39309699/d09c2ef2-05fe-4fa0-b8b4-11bf558d2eab"/>
+
+## Step 10: Swapping NEAR for Wrapped NEAR
+
+To swap your NEAR tokens for Wrapped NEAR, call `near_deposit` and deposit the desired amount of NEAR.
+
+```bash
+near call wrap.testnet near_deposit --args '{}' --accountId $CALLER_ACCOUNT_ID --amount 10 --gas 300000000000000
+```
+
+## Viewing the balance on NEAR testnet
+
+After swapping your tokens, you can check your balance on the NEAR Testnet using the following method:
+
+```bash
+near view wrap.testnet ft_balance_of --args '{"account_id": "$REGISTERED_ACCOUNT_ID"}'
+```
+
+## Viewing the balance in the FT Connector
+
+You can also check your balance in the FT Connector by running the following command:
+
+```bash
+near view wrap.testnet ft_balance_of --args '{"account_id": "ft_connector.$SHARD_ID.calimero.testnet"}'
+```
+
+## Step 11: Sending Wrapped NEAR to the FT Connector
+
+When sending Wrapped NEAR to the FT Connector, the NEAR account acts as a locker on the NEAR side, while on the Calimero side, the account `ft_deployer.` is responsible for minting and distributing tokens to accounts in the shard.
+
+You can check the list of all transactions in the [console explorer](https://app.calimero.network/explorer/dashboard).
+
+
+```bash
+near call wrap.testnet ft_transfer_call --args '{"receiver_id": "ft_connector.$SHARD_ID", "amount": "1", "msg": ""}' 
+--accountId $CALLER_ACCOUNT_ID --depositYocto 1 --gas 300000000000000
+```
+
+## Step 12: Viewing the FT balance in Calimero Shard
+
+You can check the balance of your account in the Calimero Shard using `wrap.ft_deployer`, which represents the `wrap.testnet` fungible token on the Calimero side.
+
+```bash
+near view wrap.ft_deployer.$SHARD_ID.calimero.testnet ft_balance_of --args '{"account_id": "$REGISTERED_ACCOUNT_ID"}' \
```

Comment:
Why are we not using the Calimero alias here?
Link: https://docs.calimero.network/bridge/bridging/fungible_tokens#helper-cli

With that, this command would be simpler and shorter because we can omit --nodeUrl and --networkId part